fastkafka_gen/cli.py

```diff
@@ -66,14 +69,40 @@ def _strip_white_spaces(description: str) -> str:
     return pattern.sub(" ", description).strip()
 
 # %% ../nbs/CLI.ipynb 13
+EMPTY_DESCRIPTION_ERROR = "Error: you need to provide the application description by providing it with the --description argument or by providing it within a textual file wit the --input_file argument."
```

Comment:
Please remove --description from the help message. CLI arguments don't have alias name.